_ml-commons-plugin/algorithms.md

```diff
@@ -316,4 +400,218 @@ The API responds with the sum of the contribution and base values per aggregatio
 
 ### Limitations
 
-The Localization algorithm can only be executed directly. Therefore, it cannot be used with the ML Commons Train and Predict APIs.
\ No newline at end of file
+The Localization algorithm can only be executed directly. Therefore, it cannot be used with the ML Commons Train and Predict APIs.
+
+## Logistic regression
+
+A classification algorithm, logistic regression models the probability of a discrete outcome given an input variable. In ML commons, these classifications include both binary and multi-class. The most common is the binary classification, which takes two values such as "true/false" or "yes/no", and predicts the outcome based on the values specified. On the other hand, a multi-class output can categorize different inputs based on type. This makes logistic regression most useful for situations where you are trying to determine how your inputs fit best into a specified-category. 
+
+### Parameters
+
+| Parameter | Type | Description | Default Value |
+|---|---|---|---|
+| learningRate | Double | The rate of speed the gradient moves during gradient descent | 1 |
+| momentumFactor | Double | The idea that a regressor that had recently risen or fallen during convergence will continue that trend over the medium term | 0 |
+| epsilon | Double | The criteria that a linear model is identified | 0.1 |
```

Comment:
Does this description mean one of these two:
"The criteria that determines whether a linear model is identified"?
"The criteria that identifies a linear model"?

Reply:
Epsilon normally denotes the error... Is this the case here?

Reply:
The momentum factor is a value between 0 and 1, and it is an extra weight added onto the weight factors that accelerates the rate at which the weight factors are adjusted which helps move the minimization routine out of local minima.
The epsilon is the value for stabilising the gradient inversion.

Reply:
Thank you!